Lombiq.OrchardCoreApiClient/Models/EditApiViewModel.cs

```diff
@@ -0,0 +1,19 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace Lombiq.OrchardCoreApiClient.Models;
+
+public class EditApiViewModel
```

Comment:
Both of these are the same model, and you can rename the single class to `TenantApiModel` to adopt to what OC does now.

Reply:
I see, so should I just create a `TenantApiModel` and use that for both creating and editing the tenant, and leave the `SetupApiViewModel` as it is?
Based on it has other properties, and the `TenantApiModel` will have properties which shouldn't be part of the `SetupApiViewModel`.

Reply:
Yep, setup is different. But let's also rename that to `TenantSetupApiModel`.